{
  "id": "CVE-2023-25399",
  "published": "2023-07-05T17:15:09",
  "modified": "2023-11-29T09:54:34.973839",
  "details": "A refcounting issue which leads to potential memory leak was discovered in scipy commit 8627df31ab in Py_FindObjects() function.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "repo_url": "https://github.com/scipy/scipy",
  "cwes": [],
  "commits": [
    "496efea2f16ea680f9d3a1ad066dfc6a093f6372"
  ]
}
